package.json

```diff
@@ -16,7 +16,8 @@
     "test:infrastructure": "jest test/tests/infrastructure",
     "test:services": "jest test/tests/services",
     "test": "jest && codecov",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
```

Comment:
Yes, you are right. 
I added the "prepare" stage only to install the library from my repository, and test it in my project.
If you merge in master, we can also remove this stage, because during the publishing on npm, the build stage will be done.